config/tool_conf.xml.sample

```diff
@@ -1,5 +1,5 @@
 <?xml version='1.0' encoding='utf-8'?>
-<toolbox>
+<toolbox monitor="true">
```

Comment:
Actually it does work for me, I think I thought I replicated it but was looking at tool_conf.xml instead of the sample.